Core/Graphics/Source/Texture.cpp

```diff
@@ -1,6 +1,22 @@
 #include "Texture.h"
 #include "DeviceContext.h"
 #include <cassert>
+#include <bimg/bimg.h>
+
+namespace
+{
+    const bgfx::Memory* GetZeroImageMemory(uint16_t width, uint16_t height, bool hasMips, uint16_t numLayers, bgfx::TextureFormat::Enum format)
+    {
+        bgfx::ReleaseFn releaseFn{[](void*, void* userData) {
+            bimg::imageFree(static_cast<bimg::ImageContainer*>(userData));
+        }};
+
+        bimg::ImageContainer* image = bimg::imageAlloc(&Babylon::Graphics::DeviceContext::GetDefaultAllocator(), static_cast<bimg::TextureFormat::Enum>(format), width, height, 1/*depth*/, numLayers, false/*cubeMap*/, hasMips);
+        const bgfx::Memory* mem{bgfx::makeRef(image->m_data, image->m_size, releaseFn, image)};
```

Comment:
nit: It's weird to use two different styles in the same place.
```suggestion
        bimg::ImageContainer* image = bimg::imageAlloc(&Babylon::Graphics::DeviceContext::GetDefaultAllocator(), static_cast<bimg::TextureFormat::Enum>(format), width, height, 1/*depth*/, numLayers, false/*cubeMap*/, hasMips);
        const bgfx::Memory* mem = bgfx::makeRef(image->m_data, image->m_size, releaseFn, image);
```

Reply:
fixed